solutions/tests/test_mean.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import unittest
```

Comment:
Adding a docstring to this test file that explains its purpose and the tests it includes would enhance clarity for reviewers